homeassistant/components/imeon_inverter/manifest.json

```
@@ -0,0 +1,12 @@
+{
+  "domain": "imeon_inverter",
+  "name": "Imeon Inverter",
+  "after_dependencies": [],
+  "codeowners": ["@Imeon-Energy"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/imeon_inverter",
+  "integration_type": "hub",
```

```suggestion
  "integration_type": "device",
```
since you input the address I suspect this is only one device at a time?